src/neo/Wallets/Wallet.cs

```diff
@@ -298,6 +301,32 @@ public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null
                                 }
                                 sb.EmitAppCall(output.AssetId, "transfer", account, output.ScriptHash, value);
                                 sb.Emit(OpCode.ASSERT);
+
+                                // Solid transfer
+
+                                if (output.SolidTransfer)
+                                {
+                                    byte[] solidTransferScript;
+                                    using (ScriptBuilder sbSolid = new ScriptBuilder())
+                                    {
+                                        sbSolid.EmitPush(value);
```

Comment:
Do we have some "default" guarantee for EmitPush value? If we can make a standard here, then it simplifies above multiple "numeric matches". Maybe just adopt some large push alternative, which is enough for most people. Witnesses can be pruned on Neo3, so this won't cause big damage.